.github/workflows/lint.yaml

```diff
@@ -14,3 +14,24 @@ jobs:
         uses: frenck/action-yamllint@v1
         with:
           path: catalog/
+  tknparse:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout files
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
```

Comment:
I actually wonder if we need this at all. The comment in the other workflow said that 2 is the minimum. But it turns out that the checkout action itself has a default value of 1:
https://github.com/marketplace/actions/checkout
```
# Number of commits to fetch. 0 indicates all history for all branches and tags.
    # Default: 1
    fetch-depth: ''
```

From the changed-files action docs it's not clear to me when you need more than one:
https://github.com/marketplace/actions/changed-files
> IMPORTANT: For push events when configuring [actions/checkout](https://github.com/actions/checkout#usage) the fetch-depth should be set to either fetch-depth: 0 OR fetch-depth: 2 depending on your use case.

It's not this case - this is not a push event. But even for push, I wonder why you would need at least 2. Any idea? (You can be merging several commits at once...)

In the changed-files docs, they point to some examples. And for example here they have both PR and push events and they don't specify the depth: https://github.com/coder/code-server/blob/c32a31d802f679846876b8ad9aacff6cf7b5361d/.github/workflows/build.yaml#L48

Same here: https://github.com/qgis/QGIS/blob/a5333497e90ac9de4ca70463d8e0b64c3f294d63/.github/workflows/code_layout.yml#L147

Anyway, just some food for thought. I don't think it matters much, the penalty for fetching the extra (previous) commit is likely very small. Feel free to leave it as it is.